library/sanitize.inc.php

```diff
@@ -32,4 +32,28 @@ function convert_safe_file_dir_name($label) {
   return preg_replace('/[^A-Za-z0-9_.-]/','_',$label);
 }
 
+//image mime check with all image formats.
+function image_has_right_mime($image_properties) {
+  $mime = $image_properties["mime"];
+    $mime_types = array('image/png',
+                            'image/jpeg',
+                            'image/gif',
```

Comment:
And they get to be stated here too. @prondubuisi is the `$extension` actually used?

Reply:
Oh never mind i get it now. Good job MIME stuff and extension stuff. But could this not be the same thing... since the MIME validates the extension too just wondering 🤔